grpc.go

```diff
@@ -274,7 +277,7 @@ func (a *agentGRPC) runProcess(cid string, agentProcess *pb.Process) (pid int, e
 	defer a.sandbox.subreaper.RUnlock()
 
 	if err := ctr.container.Run(&(proc.process)); err != nil {
-		return -1, fmt.Errorf("Could not run process: %v", err)
+		return -1, status.Errorf(grpc.Code(err), "Could not run process: %v", err)
```

Comment:
http://127.0.0.1:6060/pkg/google.golang.org/grpc/#Code says grpc.Code() is deprecated.
But it still seems that grpc.Code() is fit here.